packages/expo-updates/ios/EXUpdates/AppLoader/EXUpdatesAppLoaderTask.m

```diff
@@ -210,7 +210,7 @@ - (void)_loadEmbeddedUpdateWithCompletion:(void (^)(void))completion
             [self->_selectionPolicy shouldLoadNewUpdate:[EXUpdatesEmbeddedAppLoader embeddedManifestWithConfig:self->_config database:self->_database]
                                      withLaunchedUpdate:launchableUpdate
                                                 filters:manifestFilters]) {
-          self->_embeddedAppLoader = [[EXUpdatesEmbeddedAppLoader alloc] initWithConfig:self->_config database:self->_database directory:self->_directory completionQueue:self->_loaderTaskQueue];
+          self->_embeddedAppLoader = [[EXUpdatesEmbeddedAppLoader alloc] initWithConfig:self->_config database:self->_database directory:self->_directory launchedUpdate:nil completionQueue:self->_loaderTaskQueue];
```

Comment:
Is this nil because we're going to load an embedded update w/out sending an HTTP request and so the value of launchedUpdate doesn't matter?

Reply:
Yes, that's correct. I'll add a comment here clarifying this